blocks/redirects-formatter/redirects-formatter.js

```diff
@@ -29,7 +29,7 @@ export function parseUrlString(input) {
   const pairs = input.split('\n');
 
   return pairs.reduce((rdx, pairString) => {
-    const pair = pairString.split('\t');
+    const pair = pairString.split('\t').length > 1 ? pairString.split('\t') : pairString.split(' ');
```

Comment:
Nit, could split tabs and spaces instead of checking
```suggestion
    const pair = pairString.split(/\t| /);
```